test/miner_ct_utils.erl

```diff
@@ -772,6 +772,8 @@ init_per_testcase(Mod, TestCase, Config0) ->
     NumGateways = proplists:get_value(num_gateways, Config, get_config("T", 8)),
     NumConsensusMembers = proplists:get_value(num_consensus_members, Config, get_config("N", 7)),
     LoadChainOnGateways = proplists:get_value(gateways_run_chain, Config, true),
+    GwMuxEnable = proplists:get_value(gateway_and_mux_enable, Config, false),
+    DefaultKeys = proplists:get_value(default_keys, Config, false),
```

Comment:
see below where this is consumed; since the miner will auto-generate file-based ecc_compact keys if one is not supplied and i couldn't find any instances where we needed to _specify_ keys for a test (we generate them randomly and inject them directly into the `blockchain, keys` appenv) adding this switch to allow transitioning tests to generate their own keys as part of the miners' startup felt like a non-intrusive way to deprecate that code. this would allow removal of a code path here https://github.com/helium/miner/blob/5aef195d561778d2f8e39a4685045b0f5bd512f7/src/miner_keys.erl#L194 that is only used for testing but does _not_ work when `gateway_and_mux_enable` is true